### ISSUE TITLE
Fernandez Huerta failing 'empty string' test

### ISSUE THREAD

**Issue by alxwrd** [shivam5992/textstat#155]
Fernandez Huerta test expects an empty string to score 0.0, when it should score the max score of 206.84.

- Update `test_fernandez_huerta` to `assert score == 206.84`.
- Update `fernandez_huerta` to round to 2 places.

```plain
________________________________________________ test_fernandez_huerta _________________________________________________

    def test_fernandez_huerta():
        textstat.set_lang("es")
        score = textstat.fernandez_huerta(long_spanish_text)

        assert score == 43.1

        score = textstat.fernandez_huerta(empty_str)

>       assert score == 0.0
E       assert 206.9 == 0.0

test.py:445: AssertionError
```

**Comment by Clickedbigfoot** [comment]
Are we sure this is a bug? The scale would consider a score of 0 as supremely difficult to read and an empty string shouldn't be interpreted as that.
For reference, https://legible.es/blog/lecturabilidad-fernandez-huerta/ explicitly labels scores 0-30 as university level. A perfect score of 206.9 (The rounded constant in the equation) would be interpreted as a text devoid of any difficulty whatsoever.

**Comment by alxwrd** [comment]
> Perhaps the test case should be changed instead to assert score == 206.9

I agree given _"0-30 | muy difícil"_. I will update the issue body.

**Comment by Clickedbigfoot** [comment]
Changes were made with pull request #165 . The fernandez_huerta function in textstat was also corrected with 206.85 changed to 206.84

**Comment by alxwrd** [comment]
Closed with #165
